src/components/CallHistoryLastUpdatedStatus.tsx

```diff
@@ -18,14 +18,24 @@ const CallHistoryLastUpdatedStatus = (): ReactElement => {
         });
     }, []);
 
+    const DisplayResult = () => {
+        const date = mirBstDateFormatter(reportStatus, true);
+        if (date == "Invalid Date") {
+            return date;
+        }
+        return ` (${date})`;
+    };
+
     const ReportStatusText = () => {
         if (reportStatusFailed) {
             return (<>Unknown</>);
         }
         return (
             <>
                 {<TimeAgo live={false} date={reportStatus}/>}
-                {(reportStatus ? "" + dateFormatter(reportStatus).tz("Europe/London").format(" (DD/MM/YYYY HH:mm:ss)") : "Loading")}
+                {(reportStatus ?
```

Comment:
The parentheses are not required inside the braces.

ps. I think this would look nicer one one line.